backend/src/main/webapp/blackswan/script.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const FLAG_LOCAL = 1;
+/** 
+ * When testing locally, need to append host that python service 
+ * runs on to requests.
+ */
+const LOCAL = FLAG_LOCAL === 1 ? 'http://localhost:8889' : '';
```

Comment:
nit: maybe instead of 'LOCAL' you can call it 'HOST_PREFIX' or something similar. calling it `LOCAL` makes it seem as though it is a flag specifically for local runs

Reply:
yep, I will fix this! `HOST_PREFIX` sounds like a much better name :) 

thanks!